store/revisions.js

```diff
@@ -118,7 +118,10 @@ export const actions = {
     let _timestamps = apiPage.map(item => item.timestamp);
     commit(`updateTimestamps`, _timestamps);
     apiPage.forEach(item => {
-      if (!(item.wikiRevId in state.wikiRevIdToMeta)) {
+          // skip if already in list:
+      if (!(item.wikiRevId in state.wikiRevIdToMeta)
+          // skip reviewing one's own edit:
+          && ((rootState.user.profile || {}).displayName != item.user)) {
```

Comment:
There are two ways that user matches.
- If users are not logged in. it's using `userGaId`, a per-session cookie generated by Google Anaytics
- If users are logged in, it's using the `rootState.user.profile`.

You might want to handle the `userGaId` as well.

Reply:
@epicfaace Ashwin, I am very happy that you are back to join the review.

Reply:
@xinbenlv  if the user isn't logged in, we don't know their wikipedia username, therefore we can't match anything, right?
if we try to match userGaId vs item.user , it will always be different because the userGaId won't equal the recorded user from Wikipedia API. So the only case when we can do this is when the user is logged in. Let me know if I'm missing something.

Reply:
That's right. Currently it's majorly based on `userGaId`, but in the future we want them to mostly based on `wikiUserName` as the primary way, but fall back to userGaId if not logged in.